config/application.rb

```diff
@@ -50,24 +47,5 @@ class Application < Rails::Application
 
     # Force SSL for all connections in single-mode
     config.force_ssl = true if ENV['FORCE_SSL'] == 'true'
-
-    config.middleware.insert_before 0, 'Rack::Cors' do
```

Comment:
They aren't deleted but moved to a new file `/config/initializers/cors.rb` which is the default in Rails 5.